.github/workflows/maintainer_merge_review.yml

```diff
@@ -2,7 +2,7 @@ name: Maintainer merge (review)
 
 on:
   pull_request_review_comment:
```

Comment:
```suggestion
  pull_request_review:
```
I'm going to switch this file to pull_request_review and possibly add a new file for pull_request_review_comment.